packages/docs/tutorial/tutorial-additional-page.yaml

```diff
@@ -0,0 +1,213 @@
+# Copyright 2020-2022 Lowdefy, Inc
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+_ref:
+  path: templates/general.yaml.njk
+  vars:
+    pageId: tutorial-additional-page
+    pageTitle: 4. Additional Page
```

Comment:
Maybe this could be something like "View Page". I don't know, additional page feels a little weird. What do you think?

Reply:
Maybe "Data view page" or "Data display page" or something

Reply:
Change made in fafa178269045db9fe30d99b804558ff83657243